tests/test_project.py

```diff
@@ -825,8 +924,18 @@ def get_sp(i):
                 'c': i % 3
             }
 
+        def get_doc(i):
+            i += 1
+            return {
+                'a': i,
+                'b': i % 2,
+                'c': i % 3
+            }
+
         for i in range(12):
-            self.project.open_job(get_sp(i)).init()
+            job = self.project.open_job(get_sp(i))
+            job.init()
```

Comment:
```suggestion
            job = self.project.open_job(get_sp(i)).init()
```

Reply:
Done

Reply:
`self.project.open_job(get_sp(i)).init()` return `None`. So I am reverting the change.

Reply:
That should absolutely not be the case:
https://github.com/glotzerlab/signac/blob/cbce7ccace9cc006157845d1128b0619bfac23e9/signac/contrib/job.py#L401

Reply:
I got the following error.
![error](https://user-images.githubusercontent.com/46069089/81961602-36c99a80-9630-11ea-834d-66441528bc5a.PNG)

Reply:
@vishav1771 Could you please investigate the reason for this? Are we wrapping the `Project` class in the tests implementation or so? As I pointed out, obviously `init()` must return an instance of `Job`. If it doesn't something is very wrong.

Reply:
Sure, will look into that 👍

Reply:
Done.